gpMgmt/bin/gppylib/programs/gppkg.py

```diff
@@ -188,7 +187,7 @@ def run(self):
                 BuildGppkg(self.build, None).run()
             return
 
-        if platform.linux_distribution()[0] == 'Ubuntu':
+        if linux_distribution_id() == 'ubuntu':
```

Comment:
minor nit:
I noticed in `gpMgmt/bin/gppylib/operations/test/regress/test_package/__init__.py` we changed `==` to `dist.lower().startswith('ubuntu')`. Just throwing out the idea of consistency, but pretty minor overall.

```suggestion
        if linux_distribution_id().lower().startswith('ubuntu')':
```

Reply:
It is equivalent in any case as Ubuntu documents this to be `ubuntu`.  Leaving it as is.